recipes/13/13_views.md

```diff
@@ -0,0 +1,51 @@
+# 13 Encapsulating Logic with (Temporary) Views
+
+:bulb: This example will show how you can use (temporary) views to reuse code and to structure long queries and scripts. 
+
+`CREATE (TEMPORARY) VIEW` defines a view from a query. 
+**A view is not physically materialized.** 
+Instead, the query is run every time the view is referenced in a query.
+
+Temporary views are very useful to structure and decompose more complicated queries and to re-use queries within a longer script.
+Non-temporary views - stored in a persistent catalog - can also be used to share common queries within your organization, e.g. common filters or pre-processing steps.  
```

Comment:
Maybe link to the catalog documentation? We haven't introduced catalogs yet.

Reply:
Done.